shared/src/main/kotlin/com/egm/stellio/shared/queryparameter/OptionsValue.kt

```diff
@@ -0,0 +1,8 @@
+package com.egm.stellio.shared.queryparameter
+
+enum class OptionsValue(val value: String) {
+    SYS_ATTRS("sysAttrs"),
+    KEY_VALUES("keyValues"),
+    NO_OVERWRITE("noOverwrite"),
+    OBSERVED_AT("observedAt")
```

Comment:
`observatedAt` is a specific query parameter (and not a possible value for `options`)